frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseOverrideForkPointAction.java

```diff
@@ -122,7 +122,7 @@ private void setOverrideForkPointConfigValues(
     }
 
     try {
-      GitConfigUtil.setValue(project, root, whileDescendantOfKey, ancestorCommit.getHash());
+      GitConfigUtil.setValue(project, root, whileDescendantOfKey, forkPoint.getHash());
```

Comment:
This is not strictly related to the change at hand, it's a step towards getting rid of `whileDescendantOf` completely (https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1030)